lte/gateway/c/oai/tasks/nas/esm/DedicatedEpsBearerContextActivation.c

```diff
@@ -382,7 +382,9 @@ int esm_proc_dedicated_eps_bearer_context_reject(
       /*
        * Failed to release the dedicated EPS bearer context
        */
-      *esm_cause = ESM_CAUSE_PROTOCOL_ERROR;
+      if (esm_cause) {
+        *esm_cause = ESM_CAUSE_PROTOCOL_ERROR;
+      }
```

Comment:
Why do we check here? Is there a case where caller is not allocating the memory?

Reply:
There is no significance of esm_cause, I will make changes accordingly